datadog/resource_datadog_dashboard.go

```diff
@@ -5009,6 +5025,376 @@ func buildTerraformTimeseriesDefinition(datadogDefinition datadogV1.TimeseriesWi
 	return terraformDefinition
 }
 
+//
+// Sunburst Widget Definition helpers
+//
+
+func getSunburstDefinitionschema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"request": {
+			Description: "Nested block describing the request to use when displaying the widget. Multiple `request` blocks are allowed with the structure below (exactly one of `q`, `log_query` or `rum_query` is required within the `request` block).",
+			Type:        schema.TypeList,
+			Optional:    true,
+			Elem: &schema.Resource{
+				Schema: getSunburstRequestSchema(),
+			},
+		},
+		"hide_total": {
+			Description: "Whether or not to show the total value in the widget.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+		},
+		"legend_inline": {
+			Description: "Used to configure the inline legend. Cannot be used in conjunction with legend_table.",
+			Type:        schema.TypeList,
+			Optional:    true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"type": {
+						Description:      "The type of legend (table or none).",
```

Comment:
```suggestion
						Description:      "The type of legend.",
```

The auto generated docs should have the enums listed and I don't think `none` is valid in this case.

Reply:
`"none"` is a valid value for this field.